packages/medusa/src/services/order.ts

```diff
@@ -164,8 +164,9 @@ class OrderService extends TransactionBaseService {
     const orderRepo = this.manager_.getCustomRepository(this.orderRepository_)
     const query = buildQuery(selector, config)
 
-    const { select, relations, totalsToSelect } =
-      this.transformQueryForTotals(config)
+    const { select, relations, totalsToSelect } = this.transformQueryForTotals(
+      config
+    )
```

Comment:
todo: my linter wants this to be like below - can you check your setup?

```js
    const { select, relations, totalsToSelect } =
      this.transformQueryForTotals(config)
```

Reply:
Yeah, apparently ESlint and Prettier are behaving differently. I will check the rules, but I hope this is not blocking the PR for now.
At some point we have to apply a lint --fix to the entire project and validate it as the first step in the pipeline.

Reply:
👍